### PR TITLE
OPSEXP-2318 Build imagemagick packages for Rockylinux 9

### PR DESCRIPTION
**Pull request by slohe1**
OPSEXP-2318

### CODE REVIEW

File: rpms/Dockerfile
Comment (alxgomz) [inline rpms/Dockerfile:1]: Why do we need multistage build here?

Comment (gionn) [reply rpms/Dockerfile:1]: just split the single run in multiple run if the problem is that RL9 doesn't need to enable powertools to be able to install rpm-build (just make sure to keep the clean all command as the last command for every run with yum install)

**Review by gionn — CHANGES_REQUESTED** [review]

File: rpms/Dockerfile
Comment (slohe1) [reply rpms/Dockerfile:1]: Yes, Rocky9 doesn't enable powertools, that is the why used that approach as other OS needs powertools.

**Review by alxgomz — APPROVED** [review]

File: rpms/Dockerfile [hunk 1]
```diff
@@ -2,17 +2,24 @@ ARG BASE_IMAGE
...
 ENV BASE_IMAGE=${BASE_IMAGE}
 
+RUN if [ "$BASE_IMAGE" = "rockylinux:9" ]; then \
+        dnf install -y dnf-plugins-core && \
+        dnf config-manager --set-enabled crb && \
+        yum clean all; \
+    else \
+        yum-config-manager --enable powertools && \
+        yum clean all; \
+    fi
```
Comment (gionn) [inline rpms/Dockerfile:19]: I am not sure it's really necessary if we are not installing anything else later, can you try to just remove this entire block or provide a link to a previous build which explain this requirement?

**Review by gionn — CHANGES_REQUESTED** [review]

File: rpms/Dockerfile (hunk 1 above)
Comment (slohe1) [reply rpms/Dockerfile]: https://github.com/Alfresco/imagemagick-build/actions/runs/6312526842/job/17138757252

**Review by gionn — APPROVED** [review]

**Pull request merged**